lib/reporters/base.js

```diff
@@ -152,6 +152,28 @@ exports.cursor = {
 };
 
 /**
+ * This function dumps long stack traces for exceptions having a cause()
+ * method. The error classes from
+ * [verror](https://github.com/davepacheco/node-verror) and
+ * [restify v2.0](https://github.com/mcavage/node-restify) are examples.
+ *
+ * @param {Error} ex
+ * @return {string}
+ * @api public
+ */
+
+exports.getFullErrorStack = function(err, message) {
+  var ret = err.stack || message || '';
```

Comment:
hmm, must be that we won't add a non-object error to the list.  looking into that.

I hadn't noticed `err.inspect` here, which sets a precedent for this PR anyway.